azure-quantum/tests/unit/test_problem.py

```diff
@@ -107,32 +110,32 @@ def test_problem_name_serialization(self):
             assert problem_json["metadata"]["name"] == problem_name
 
             # deserializes name
-            deserialized_problem = Problem.deserialize(problem_as_json=serialized_problem)
+            deserialized_problem = Problem.deserialize(problem_str=serialized_problem)
```

Comment:
the fact that you're changing the parameter name is a breaking change. You should add a new parameter and keep the old one.

Reply:
Oh right, got it! I'll update